src/flask/cli.py

```diff
@@ -48,7 +48,13 @@ def find_best_app(script_info, module):
             return app
 
     # Otherwise find the only object that is a Flask instance.
-    matches = [v for v in module.__dict__.values() if isinstance(v, Flask)]
+    matches = []
+    for v in module.__dict__.values():
+        try:
+            if isinstance(v, Flask):
+                matches.append(v)
+        except Exception:
+            pass
```

Comment:
Please, handle the possible exceptions because unless you handle them both explicitly or implicitly are meaningless.